File: Br.Com.Fiap.Gere.Residuo/Model/MoradorModel.cs
namespace Br.Com.Fiap.Gere.Residuo.Model
{
    public class MoradorModel
    {
        public int MoradorId { get; set; }

        // Relacionamento com BairroModel
        public int BairroId { get; set; }
        public BairroModel BairroDoMorador { get; set; }

        public string MoradorNome { get; set; }

        public string MoradorEmail { get; set; }
    }
}

File: BDD_Tests/BDD-Gere-Residuo/src/test/java/model/morador/MoradorModel.java
package model.morador;

import com.google.gson.annotations.Expose;
import lombok.Data;

@Data
public class MoradorModel {

    @Expose(serialize = false)
    private int moradorId;
    @Expose
    private int bairroId;
    @Expose
    private String moradorNome;
    @Expose
    private String moradorEmail;
}
